data-serving/data-service/api/openapi.yaml

```diff
@@ -468,6 +494,13 @@ components:
           type: array
           items:
             type: string
+    PlacesOfTransmissionArray:
+      type: object
+      properties:
+        symptoms:
```

Comment:
s/symptoms/placesOfTransmission?

I'm a little surprised this didn't throw an error in the tests

Reply:
Me too... strange. Done.